### ISSUE TITLE
Tact imports don't work 

### ISSUE THREAD

**Issue by Cosmodude** [ton-org/blueprint#51]
code: 
```import "@stdlib/deploy";
import "@stdlib/ownable";
import "@stdlib/stoppable";
import "./imports/messages";

const day: Int = (24 * 60 * 60);
const year: Int =  24 * 60 * 60 * 365;
...
```

When running

``npx blueprint build``

Get this error: 
Error: Could not resolve import ./imports/messages 
I think the problem is with the blueprint framework since file exists and is located properly

**Comment by krigga** [comment]
Fixed in v0.12.0, please reopen if issue persists

**Comment by defi-pickle** [comment]
relative imports don't work. `Error: Could not resolve import ../imports/messages.tact`
When i have given structure: 
```
root/
--imports/
----messages.tact
--jetton/
----jetton_master.tact
```
and with this compile configs: 
```

export const compile: CompilerConfig = {
    lang: 'tact',
    target: 'root/jetton/jetton_master.tact',
    options: {
        debug: false,
    },
};
```
